### PR TITLE
Updating sizing and positioning for group 5 images

### PR DESCRIPTION
**Pull request by chrisdmills-okta**
## Description:
- **What's changed?** The PR reorganizes the images in Group 5 (as seen in https://docs.google.com/spreadsheets/d/1M6zdk6XVAnj7uBoEbVvRpBjeFjB_32lFyN-AombSB50/edit#gid=0), according to the plan written at https://oktainc.atlassian.net/browse/OKTA-511070.
- **Is this PR related to a Monolith release?** No

### Resolves:

* [OKTA-511077](https://oktainc.atlassian.net/browse/OKTA-511077)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-custom-pwd-recovery-mfa/main/java/siw.md [hunk 1]
```diff
@@ -109,7 +109,7 @@ Pass the values using the widget's `config` object.
 
 After the widget loads, it checks the validity of the `otp` and `state` values with the Okta server. If they are valid, the user is prompted to reset their password, as shown below. The user continues the password recovery flow described in the [User password recovery summary of steps](/docs/guides/oie-embedded-sdk-use-case-pwd-recovery-mfa/java/main/#summary-of-steps).
 
-<div class="common-image-format bordered-image">
```
Comment (danielmaharry-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-custom-pwd-recovery-mfa/main/java/siw.md:112]: CSS border doesn't match the rounded border radius of the image.

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-custom-pwd-recovery-mfa/main/java/sdk.md [hunk 2]
```diff
@@ -104,8 +104,8 @@ public ModelAndView handleKnownTransitions(
 
 Display the password reset page and continue the password recovery flow described in the [User password recovery summary of steps](/docs/guides/oie-embedded-sdk-use-case-pwd-recovery-mfa/java/main/#summary-of-steps).
 
-<div class="common-image-format">
```
Comment (danielmaharry-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-custom-pwd-recovery-mfa/main/java/sdk.md:107]: Image has got some drop-shadows going on in it. Need to remove those or wait for wireframe?

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-custom-pwd-recovery-mfa/main/nodeexpress/siw.md [hunk 3]
```diff
@@ -88,7 +88,7 @@ After set up, render the client site page and pass the configurations to the wid
 
 After the widget is loaded, the following reset page is appears:
 
-<div class="common-image-format bordered-image">
```
Comment (danielmaharry-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-custom-pwd-recovery-mfa/main/nodeexpress/siw.md:91]: CSS border doesn't match the rounded border radius of the image.

**Review by danielmaharry-okta — CHANGES_REQUESTED** [review]
Just a few borders to remove and one bit of text to change

File: packages/@okta/vuepress-site/docs/guides/authenticators-google-authenticator/main/nodeexpress/enrollmentintegrationsteps.md [hunk 4]
```diff
@@ -46,7 +46,7 @@ Use the `value` and `label` properties to show the available list of authenticat
 
 UI showing the Google Authenticator option:
 
-<div class="common-image-format">
```
[the remark this answers is not in the document]
Comment (chrisdmills-okta) [reply packages/@okta/vuepress-site/docs/guides/authenticators-google-authenticator/main/nodeexpress/enrollmentintegrationsteps.md:49]: Border removed.

File: packages/@okta/vuepress-site/docs/guides/authenticators-google-authenticator/main/nodeexpress/challengeintegrationsteps.md [hunk 5]
```diff
@@ -62,7 +62,7 @@ Use the `IdxTransaction` properties to show a challenge page that displays a one
 
 UI showing the one-time password input field.
 
-<div class="common-image-format">
```
[the remark this answers is not in the document]
Comment (chrisdmills-okta) [reply packages/@okta/vuepress-site/docs/guides/authenticators-google-authenticator/main/nodeexpress/challengeintegrationsteps.md:65]: border removed

File: packages/@okta/vuepress-site/docs/guides/authenticators-google-authenticator/main/nodeexpress/challengeintegrationsteps.md [hunk 6]
```diff
@@ -72,7 +72,7 @@ UI showing the one-time password input field.
 
 Next, the user opens Google Authenticator on their mobile device and finds the one-time password for their account.
 
-<div class="common-image-format">
```
[the remark this answers is not in the document]
Comment (chrisdmills-okta) [reply packages/@okta/vuepress-site/docs/guides/authenticators-google-authenticator/main/nodeexpress/challengeintegrationsteps.md:75]: border removed

File: packages/@okta/vuepress-site/docs/guides/authenticators-web-authn/main/aspnet/enrollmentintegrationsteps.md [hunk 7]
```diff
@@ -141,7 +141,7 @@ The `viewModel` parameter is then consumed in a Razor page.
 
 The WebAuthn factor option is listed as **Security Key or Biometric**, as shown in the following image.
 
-<div class="common-image-format bordered-image">
```
[the remark this answers is not in the document]
Comment (chrisdmills-okta) [reply packages/@okta/vuepress-site/docs/guides/authenticators-web-authn/main/aspnet/enrollmentintegrationsteps.md:144]: border removed

File: packages/@okta/vuepress-site/docs/guides/authenticators-web-authn/main/nodeexpress/enrollmentintegrationsteps.md [hunk 8]
```diff
@@ -45,7 +45,7 @@ Use the `value` and `label` properties to show the available list of authenticat
 
 UI showing the Google Authenticator option:
```
[the remark this answers is not in the document]
Comment (chrisdmills-okta) [reply packages/@okta/vuepress-site/docs/guides/authenticators-web-authn/main/nodeexpress/enrollmentintegrationsteps.md:46]: LOL, fixed ;-)

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-custom-pwd-recovery-mfa/main/java/siw.md (hunk 1 above)
Comment (chrisdmills-okta) [reply packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-custom-pwd-recovery-mfa/main/java/siw.md:112]: Intrinsic border removed

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-custom-pwd-recovery-mfa/main/aspnet/sdk.md [hunk 9]
```diff
@@ -72,8 +72,8 @@ If the `otp` and `state` values aren't valid or an `AuthenticationStatus` is ret
 
 Display the password reset page and continue the password recovery flow described in the [User password recovery guide](/docs/guides/oie-embedded-sdk-use-case-pwd-recovery-mfa/aspnet/main/).
 
-<div class="common-image-format bordered-image">
```
[the remark this answers is not in the document]
Comment (chrisdmills-okta) [reply packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-custom-pwd-recovery-mfa/main/aspnet/sdk.md:75]: intrinsic border and weird transparent right hand gutter removed.

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-custom-pwd-recovery-mfa/main/java/sdk.md (hunk 2 above)
Comment (chrisdmills-okta) [reply packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-custom-pwd-recovery-mfa/main/java/sdk.md:107]: drop shadow removed and CSS border added. Looks pretty decent now.

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-custom-pwd-recovery-mfa/main/nodeexpress/siw.md (hunk 3 above)
Comment (chrisdmills-okta) [reply packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-custom-pwd-recovery-mfa/main/nodeexpress/siw.md:91]: intrinsic border removed

File: packages/@okta/vuepress-site/docs/guides/authenticators-google-authenticator/main/nodeexpress/enrollmentintegrationsteps.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (danielmaharry-okta) [reply packages/@okta/vuepress-site/docs/guides/authenticators-google-authenticator/main/nodeexpress/enrollmentintegrationsteps.md:49]: yup

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-custom-pwd-recovery-mfa/main/aspnet/siw.md [hunk 10]
```diff
@@ -81,7 +81,7 @@ Consume the session `state` and `otp` values in a Razor page that contains the S
 
 After the widget is loaded, it checks whether the state and OTP are valid with Okta. Assuming they are, either the following reset page appears, or a prompt appears for the user to enter the OTP code. After the user enters the OTP code, the reset page appears. The user continues the password recovery flow described in the [User password recovery guide](/docs/guides/oie-embedded-sdk-use-case-pwd-recovery-mfa/aspnet/main/).
 
-<div class="common-image-format bordered-image">
```
[the remark this answers is not in the document]
Comment (danielmaharry-okta) [reply packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-custom-pwd-recovery-mfa/main/aspnet/siw.md:84]: fixed

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-custom-pwd-recovery-mfa/main/aspnet/sdk.md (hunk 9 above)
[the remark this answers is not in the document]
Comment (danielmaharry-okta) [reply packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-custom-pwd-recovery-mfa/main/aspnet/sdk.md:75]: fixed

**Comment by chrisdmills-okta** [comment]
@danielmaharry-okta cheers. Again, can you fix this merge conflict? I think it has to do with your EML overview additions.

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-custom-pwd-recovery-mfa/main/java/sdk.md (hunk 2 above)
Comment (danielmaharry-okta) [reply packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-custom-pwd-recovery-mfa/main/java/sdk.md:107]: very nice

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-custom-pwd-recovery-mfa/main/nodeexpress/siw.md (hunk 3 above)
Comment (danielmaharry-okta) [reply packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-custom-pwd-recovery-mfa/main/nodeexpress/siw.md:91]: fixed

**Review by danielmaharry-okta — APPROVED** [review]

**Pull request merged**